app/build.gradle

```diff
@@ -32,17 +41,20 @@ dependencies {
     implementation project(":featuremodule")
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'androidx.core:core-ktx:1.2.0'
+    implementation 'androidx.core:core-ktx:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0'
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
+    debugImplementation 'androidx.fragment:fragment-testing:1.2.4'
+
 
-    implementation 'com.google.dagger:dagger:2.28'
```

Comment:
with hilt you actually don't need to depend on dagger or dagger-compiler itself anymore.